content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md

```diff
@@ -5,43 +5,51 @@ page_title: "SAML Okta Identity Provider Configuration - Terraform Enterprise"
-
-5. Still in the "Configure SAML" page, configure a group attribute statement to report which teams a user belongs to. Under the "Group Attribute Statements (Optional)" header, configure the statement as follows:
-    - **Name:** `MemberOf` (This is the default name for TFE's group [attribute][]; the name of this attribute can be changed in [TFE's SAML settings](./configuration.html) if necessary.)
-    - **Name format** (drop-down): Basic
-    - **Filter:** A filter type and filter value that will match all of the relevant groups that each user belongs to. The exact filter expression depends on how your Okta groups are configured, and which subset of groups you want to expose to TFE. Note that TFE ignores group names that do not correspond to existing TFE teams; see [Team Membership Mapping](./team-membership.html) for more details.
-
-      ![Screenshot: Further down on Okta's "Configure SAML" page, with a group attribute statement partially configured.](./images/sso-okta-new-application-attributes.png)
-6. Still in the "Configure SAML" page, optionally configure a site admin permissions attribute statement. This statement determines which users can administer the entire Terraform Enterprise instance (see [Administering Terraform Enterprise](../admin/index.html) for more information about site admin permissions). Under the "Attribute Statements (Optional)" header, configure a statement as follows:
-    - **Name:** `SiteAdmin` (this is the default name for TFE's site admin [attribute][]; the name of this attribute can be changed in [TFE's SAML settings](./configuration.html) if necessary.)
-    - **Name Format** (drop-down): Basic
-    - **Value:** An [Okta expression](https://developer.okta.com/reference/okta_expression_language/) that will evaluate to a boolean: `true` for every user who should have site admin permissions, but `false` for any users who should **not** have site admin permissions. The exact expression depends on the user properties you use to manage admin permissions.
-7. Preview the SAML response and make sure it matches your expectations.
-8. Finish configuring the SAML app in Okta, then copy the provided endpoint URLs and certificate to your Terraform Enterprise SAML settings at `https://<TFE_HOSTNAME>/app/admin/saml` (example below). TFE requires a single sign-on URL, a single log-out URL, and a PEM (base64) encoded X.509 certificate.
-      ![Screenshot: TFE "Identity Provider Configuration" page for SAML (1 of 2), with populated example data.](./images/sso-tfe-okta-screen1.png)
-      ![Screenshot: TFE "Identity Provider Configuration" page for SAML (2 of 2), with populated example data.](./images/sso-tfe-okta-screen2.png)
+1. In Okta's web interface, go to the **Applications** tab and click **Create App Integration**.
+1. Select **SAML 2.0** as the sign on method, then click **Next**.
+1. In the **General Settings** page, enter `Terraform Enterprise`, optionally add an app logo, then click **Next**.
```

Comment:
```suggestion
1. In the **General Settings** page, enter `Terraform Enterprise`, optionally add an app logo, and then click **Next**.
```

Reply:
I like to tell users what page opens here. Is it the **SAML Configuration** page?  I do it like t his "The **SAML Configuration** page appears."  It's nice for users to get this marker and then I see you reference the page name later on in the directions.

Reply:
Kinda the SAML configuration page? you can find the steps with the screenshots in this tutorial: https://learn-git-enable-sso-tfe-saml-okta-hashicorp.vercel.app/tutorials/terraform/enable-sso-saml-tfe-okta?in=terraform/enterprise

Reply:
will commit this suggestion once you comment so it doesn't resolve this convo

Reply:
ahh okay I see what you mean - ignore me :P  I trust your judgment here!